mailpoet/lib/EmailEditor/Engine/SettingsController.php

```diff
@@ -94,26 +94,37 @@ public function getLayout(): array {
   }
 
   /**
-   * @return array{width: string, background: string, padding: array{bottom: string, left: string, right: string, top: string}}
+   * @return array{
+   *   layout: array{width: string,background: string,padding: array{bottom: string, left: string, right: string, top: string}},
+   *   colors: array{background: string},
+   *   typography: array[]
+   * }
    */
-  public function getEmailLayoutStyles(): array {
+  public function getEmailStyles(): array {
     return [
-      'width' => self::EMAIL_WIDTH,
-      'background' => self::EMAIL_LAYOUT_BACKGROUND,
-      'padding' => [
-        'bottom' => self::FLEX_GAP,
-        'left' => self::FLEX_GAP,
-        'right' => self::FLEX_GAP,
-        'top' => self::FLEX_GAP,
+      'layout' => [
+        'background' => self::EMAIL_LAYOUT_BACKGROUND,
+        'width' => self::EMAIL_WIDTH,
+        'padding' => [
+          'bottom' => self::FLEX_GAP,
+          'left' => self::FLEX_GAP,
+          'right' => self::FLEX_GAP,
+          'top' => self::FLEX_GAP,
+        ],
+      ],
+      'colors' => [
+        'background' => '#000000',
```

Comment:
What is the difference between layout.background and colors.background?

Reply:
The `layout.background` is used in the email editor wrapper, and the `colors.background` is used in the email.
![example](https://d.pr/i/rnWotD+)

I would prefer storing the layout background color in a different place, but I haven't found the right one yet.